### DISCUSSION TITLE
Ladyada talks about -DD series

### DISCUSSION THREAD

**Discussion by obdevel** [SpenceKonde/DxCore#337] [General]
The latest NPI video from Adafruit talks about the -DD series: https://www.youtube.com/watch?v=mKM-bcz8UtM and mentions this core.

No mention of boards from Adafruit, which will please Spence ;)

**Comment by SpenceKonde** [comment]
Now if only I could get some product descriptions ready for them and list em, I could make bank! Oh, and I guess the core needs to actually work now that I've added them
